packages/example/.storybook/main.js

```diff
@@ -1,17 +1,24 @@
 module.exports = {
-  core: {
-    builder: 'webpack5',
-  },
-  features: {
-    previewCsfV3: true,
-  },
+  // core: {
+  //   builder: {
+  //     name: '@storybook/builder-webpack5',
+  //     options: {
+  //       fsCache: true,
+  //       lazyCompilation: true,
+  //     },
+  //   },
+  // },
+  framework: '@muban/storybook-framework-webpack',
   stories: [
     {
-      directory: '../src',
-      files: '*.stories.@(js|jsx|ts|tsx)',
+      directory: '../src/',
+      files: '**/*.stories.@(js|jsx|ts|tsx)',
       titlePrefix: '',
     },
   ],
-  addons: ['@storybook/addon-links', '@storybook/addon-essentials', '@storybook/addon-postcss'],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials' /*, '@storybook/addon-postcss'*/,
+  ],
   // staticDirs: ['../public'],
```

Comment:
```suggestion
    '@storybook/addon-essentials'
  ],
```

Remove commented out configuration